### PR TITLE
Regenerate with gtk 4.0.0

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: gdk4/src/auto/toplevel_layout.rs [hunk 1]
```diff
@@ -39,11 +40,19 @@ impl ToplevelLayout {
     }
 
     #[doc(alias = "gdk_toplevel_layout_get_fullscreen")]
-    pub fn get_fullscreen(&self) -> bool {
+    pub fn get_fullscreen(&self) -> Option<bool> {
```
Comment (bilelmoussaoui) [inline gdk4/src/auto/toplevel_layout.rs:43]: Hmm, what makes gir mis-regenerates this one? it should only return a bool per the docs https://developer.gnome.org/gdk4/stable/GdkToplevelLayout.html#gdk-toplevel-layout-get-fullscreen

File: gdk4/src/auto/toplevel_layout.rs [hunk 2]
```diff
@@ -57,10 +66,11 @@ impl ToplevelLayout {
     }
 
     #[doc(alias = "gdk_toplevel_layout_get_maximized")]
-    pub fn get_maximized(&self) -> bool {
+    pub fn get_maximized(&self, maximize: bool) -> bool {
```
Comment (sdroege) [inline gdk4/src/auto/toplevel_layout.rs:69]: This is wrong. `maximize` in the C API is a pointer to a bool, it's an out parameter. Missing annotation

File: gdk4/src/auto/toplevel_layout.rs (hunk 1 above)
Comment (sdroege) [reply gdk4/src/auto/toplevel_layout.rs:43]: No, this is completely correct and the docs are outdated: https://gitlab.gnome.org/GNOME/gtk/-/blob/master/gdk/gdktoplevellayout.c#L261

File: gdk4/src/auto/toplevel_layout.rs (hunk 2 above)
Comment (sdroege) [reply gdk4/src/auto/toplevel_layout.rs:69]: Should work the same as `get_fullscreen()` basically

File: gdk4/src/auto/toplevel_layout.rs (hunk 1 above)
Comment (bilelmoussaoui) [reply gdk4/src/auto/toplevel_layout.rs:43]: Ah yes my bad. I somehow missed it's an out param.

File: gdk4/src/auto/toplevel_layout.rs (hunk 2 above)
Comment (bilelmoussaoui) [reply gdk4/src/auto/toplevel_layout.rs:69]: Seems to be fixed already in-tree, I have re-updated the gir-files in https://github.com/gtk-rs/gir-files/pull/71

**Review by sdroege — APPROVED** [review]

**Pull request merged**